gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/SocialMedia.kt

```diff
@@ -183,7 +184,7 @@ public fun SocialIconRow(socialMedia: List<SocialMedia>, modifier: Modifier = Mo
  */
 @Composable
 public fun SocialIconRow(profile: Profile, modifier: Modifier = Modifier, maxIcons: Int = 4) {
-    SocialIconRow(mediaList(profile), modifier, maxIcons)
+    SocialIconRow(mediaList(profile), modifier.offset((-5).dp), maxIcons)
```

Comment:
🤔 This helps with the icon alignment in our profile components but hardcodes the offset inside `SocialIconRow`. If someone wants to use this component isolated (or include us in a different design), they can't do anything to avoid the offset. Have you considered that?

Also, we left some "empty" space on the right of the component. I don't think this is a big deal for our Profile components, but as before, not sure if it looks good if we use it as an isolated component.

<img width="181" alt="image" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/6974554/d2dcebe8-84c5-4795-8b19-785323336241">

Reply:
>If someone wants to use this component isolated (or include us in a different design), they can't do anything to avoid the offset. Have you considered that?

I did. I've decided to include it here because the Gravatar icon is also hardcoded in that component. It's always the first element as part of of this composable (added in the `mediaList()` function). So, if the Gravatar icon is removed we can remove that offset with it. The offset can also be neutralized by setting the same positive value, so it's still possible to achieve the same effect.

Other solutions I tried:
1. Add `offset` param to `SocialMedia` class and apply it based on the icon type. This didn't work because offsetting on composable in a row didn't move others. 
2. Applying `offset` to the modifier in each profile card to the `SocialIconRow()` - the main problem here is that the same modifier is used inside this composable for the skeleton that has different paddings. So if we go this route we would have to fake the padding for the skeleton in order to match the alignment. 

I'm happy to modify this though this solution made the most sense to me. Do you have other ideas? 


>not sure if it looks good if we use it as an isolated component.

I haven't been here in the beginning so I'm not sure what's exposed on purpose and what composables are exposed as a side effect, but I thought the main goal was to provide ready-to-use, plug-and-play components - and that was my focus. I can try and modify the code to make the alternative solution number 2 work.

Reply:
oh and it's also possible to use the one function above
```public fun SocialIconRow(socialMedia: List<SocialMedia>, modifier: Modifier = Modifier, maxIcons: Int = 4)```
to have the raw list without both: the hardcoded Gravatar icon and the offset.

Reply:
> I haven't been here in the beginning so I'm not sure what's exposed on purpose and what composables are exposed as a side effectt, but I thought the main goal was to provide ready-to-use, plug-and-play components

The original idea was to provide both complex, ready-to-use components like the profile ones and the atomic components that we use to render those complex composables. That way, clients can use our profile components or create and display the information they want wherever they want, with their customized design. For that reason, the `atomic` components are exposed.

>  I can try and modify the code to make the alternative solution number 2 work

In my opinion, number 2 looks better. We'll have a skeleton identical to the view, and the offset will be set outside the component, in our case from the profile composables. It sounds good to me. 

<img width="194" alt="image" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/6974554/703443a7-1385-4395-8ab6-74c66714fef8">

<img width="411" alt="image" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/6974554/a8036b64-05cf-4d39-8f22-1c87055d2b96">

At the same time, if we use `SocialIconRow` in a different place, we don't need to neutralize the offset; it works as expected.

Wdty?

Reply:
Agree! Update the code. Let me know what you think now.